docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+
+Note that if you'd like more details than what's provided here, you can check out the [Docusaurus Docs on sidebars](https://v2.docusaurus.io/docs/sidebar).
+
+1. Create a new top-level internal link to the doc or modify an existing one
+
+<img src="../img/top-level-link.png" alt="top-level link" style={{ width: '500px' }} />
+
+To set this up, you'll have to modify the `sidebars.js` file locally or through the GH UI link above - all it needs is the `id` that is set at the top of the MD file. It will look something like this to reflect the screenshot:
+
+```js
+module.exports = {
+  sidebar: [
+    "welcome",
+    "my-flying-nimbus”, // this doc ID is all you need to add!
+    ... // other unrelated sidebar items
+```
+
+---
+
+2. Create a new top-level external link to the doc
+
+<div className="flex">
+<img src="../img/top-level-external-link.png" alt="top-level external link" style={{ width: '300px', marginRight: '32px' }} />
+
+<div>
+
+If you'd like to add a new top-level external link, <code>sidebars.js</code> requires an object expecting <code>type</code>, <code>label</code>, and <code>href</code> that will look something like this to reflect the screenshot:
+
+```js
+module.exports = {
+  sidebar: [
+     "welcome",
+   {
+     type: "link",
+     label: "My Flying Nimbus",
+     href: "https://magicalyellowcloud.com",
+   },
+   ... // other unrelated sidebar items
+```
+
+You'll want to make sure `type` is set to `"link"`.
+
+</div>
+</div>
+
+---
+
+3. Create a new internal link to the doc in an existing or new category
+
+<div className="flex">
+<div>
+
+In Docusaurusland, sidebar menu items with a down caret are called "categories."
+
+To add your internal page to an existing category, find the category via the `label` item in `sidebars.js` which will match the category text you see displayed in the sidebar (see code below). Find the `items` array for that category and add the doc ID you wish to link internally to.
+
+To add your internal page to a new category, copy and paste the entire object shown below where you'd like it to show up in the sidebar. Edit your label as desired and change `items` to `items: ["my-flying-nimbus"]` with your doc ID.
+
+</div>
+<img src="../img/category-link.png" alt="category link" style={{ width: '300px', marginLeft: '16px' }} />
+</div>
+
+```js
+module.exports = {
+  sidebar: [
+     ... // other unrelated sidebar items
+    {
+      type: "category",
+      label: "Experiment Owners", // what you're looking for, or creating
+      items: ["experiment-owners-root", "my-flying-nimbus"], // doc ID is in this array
+    },
+    ... // other unrelated sidebar items
+```
+
+---
+
+4. Create a new external link to a doc in an existing or new category
+
+<div className="flex">
+<img src="../img/category-external-link.png" alt="category link" style={{ width: '300px', marginRight: '32px', marginBottom: '16px' }} />
```

Comment:
Hrrmm, I kinda have beef with these custom spacing styles. Ideally the docs we write about how to write docs can be easily replicated, and using custom margins on images could be a barrier for someone who wants to make their contribution look like ours, and it also doesn't play well with mobile. Could these styles be added to the CSS as a class like "extra-padding" or "align-right"?